server/test/services/export/PdfExporterTest.java

```diff
@@ -30,10 +35,23 @@ public void validatePDFExport() throws IOException, DocumentException {
     int pages = pdfReader.getNumberOfPages();
     for (int i = 1; i < pages; i++) {
       textFromPDF.append(PdfTextExtractor.getTextFromPage(pdfReader, i));
+      // Assertions to check if the URL is embedded for the FileUpload
```

Comment:
Does the fake data already contain an "optional" file upload question? Can we add some assertions for how the case where no file is provided (and the program defines it as optional) is handled?

Reply:
I don't think optional or not doesn't matter to the PDFExporter. The fake data has a file upload question and I am checking the PDFAnnots to make sure the URL is added.